Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.py

```diff
@@ -188,7 +188,7 @@ def is_valid_risk_rule(client: Client, risk_rule):
         params=client.PARAMS,
         headers=client.headers
     )
-    risk_rules_list = [risk_rule['name'] for risk_rule in risk_rule_response['data']['results']]
+    risk_rules_list = [risk_rule['name'] for single_risk_rule in risk_rule_response['data']['results']]
```

Comment:
??? This won't work.

Reply:
```suggestion
    risk_rules_list = [single_risk_rule['name'] for single_risk_rule in risk_rule_response['data']['results']]
```
Just don't use `risk_rule` in your list comprehension at all since that is the name of one of your function parameters here. If you decide to call it `single_risk_rule` then you need to change it there and before.

Reply:
Of course, I forgot to also change the first one in the list comprehension.